<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>846</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Note Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="Workspace" name="workWidget" native="true"/>
    </item>
    <item row="1" column="0">
     <widget class="QWidget" name="widget" native="true"/>
    </item>
    <item row="0" column="2" rowspan="2">
     <widget class="Editorspace" name="Editeur" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>846</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuNew">
     <property name="title">
      <string>New...</string>
     </property>
     <widget class="QMenu" name="menuBinary">
      <property name="title">
       <string>Binary</string>
      </property>
      <addaction name="actionImage_2"/>
      <addaction name="actionAudio"/>
      <addaction name="actionVideo"/>
     </widget>
     <addaction name="actionWorkspace"/>
     <addaction name="actionArticle"/>
     <addaction name="actionDocument"/>
     <addaction name="menuBinary"/>
    </widget>
    <widget class="QMenu" name="menuOpen">
     <property name="title">
      <string>Open...</string>
     </property>
     <widget class="QMenu" name="menuBinary_2">
      <property name="title">
       <string>Binary</string>
      </property>
      <addaction name="actionImage"/>
      <addaction name="actionAudio_2"/>
      <addaction name="actionVideo_2"/>
     </widget>
     <addaction name="actionWorkspace_2"/>
     <addaction name="actionArticle_2"/>
     <addaction name="actionDocument_2"/>
     <addaction name="menuBinary_2"/>
    </widget>
    <addaction name="menuNew"/>
    <addaction name="menuOpen"/>
    <addaction name="actionAdd"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionAdd">
   <property name="text">
    <string>Add...</string>
   </property>
  </action>
  <action name="actionArticle">
   <property name="text">
    <string>Article</string>
   </property>
  </action>
  <action name="actionDocument">
   <property name="text">
    <string>Document</string>
   </property>
  </action>
  <action name="actionImage_2">
   <property name="text">
    <string>Image</string>
   </property>
  </action>
  <action name="actionAudio">
   <property name="text">
    <string>Audio</string>
   </property>
  </action>
  <action name="actionVideo">
   <property name="text">
    <string>Video</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As...</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionWorkspace">
   <property name="text">
    <string>Workspace</string>
   </property>
  </action>
  <action name="actionWorkspace_2">
   <property name="text">
    <string>Workspace</string>
   </property>
  </action>
  <action name="actionArticle_2">
   <property name="text">
    <string>Article</string>
   </property>
  </action>
  <action name="actionDocument_2">
   <property name="text">
    <string>Document</string>
   </property>
  </action>
  <action name="actionImage">
   <property name="text">
    <string>Image</string>
   </property>
  </action>
  <action name="actionAudio_2">
   <property name="text">
    <string>Audio</string>
   </property>
  </action>
  <action name="actionVideo_2">
   <property name="text">
    <string>Video</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Editorspace</class>
   <extends>QWidget</extends>
   <header>editorspace.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Workspace</class>
   <extends>QWidget</extends>
   <header>workspace.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
